src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -250,5 +254,69 @@ def update_confg(config_path: str, new_autovalidator_address: str, new_codename:
         raise RuntimeError(f"Failed to write to the configuration file: {config_path}.\n Error: {e}")
 
 
+def get_commands_from_server(
+    subnet_identifier: str, subnet_realm: str, wallet: bt.wallet, autovalidator_address: str
+) -> list:
+    """
+    Request commands to the server
```

Comment:
See this is exactly the reason I don't want useless docstrings and comments. They become outdated and confusing. Either have something interesting to say or don't say anything. Writing here the name of the function with underscores replaced with spaces provides no value and just is a liability. Remove it.